tile/tile-history.yml

```diff
@@ -0,0 +1,7 @@
+---
+history:
+- 0.0.1
+- 0.0.2
+- 0.0.3
```

Comment:
Should we keep theses since it's the first release of the tile ? Or do they correspond to buildpack releases ?

Reply:
I'll delete them, they're not necessary they were just tests